bundle/src/main/java/com/adobe/acs/commons/mcp/form/CheckboxComponent.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright 2017 Adobe.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.adobe.acs.commons.mcp.form;
+
+/**
+ * Radio button selector component
+ */
+public class CheckboxComponent extends FieldComponent {
```

Comment:
I hadn't looked at it from an objective lens of Consumer vs. Provider type but yeah, most of this is ProviderType with only few exceptions.  I'll update those.